src/test/java/pages/BasePage.java

```diff
@@ -15,17 +15,19 @@ public class BasePage {
     WebDriverWait wait;
     Actions actions;
 
-public BasePage( WebDriver givenDriver ){
-    driver = givenDriver;
-    wait = new WebDriverWait(driver, Duration.ofSeconds(5));
-    actions = new Actions(driver);
-}
+    public BasePage(WebDriver givenDriver) {
+        driver = givenDriver;
+        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        actions = new Actions(driver);
+    }
 
-    public WebElement waitUntilVisible(By element){
+    public WebElement waitUntilVisible(By element) {
         return new WebDriverWait(driver, Duration.ofSeconds(4)).until(ExpectedConditions.visibilityOfElementLocated(element));
     }
-
-    public WebElement waitUntilClickable(By element){
+    public WebElement waitUntilClickable (By element){
         return new WebDriverWait(driver, Duration.ofSeconds(4)).until(ExpectedConditions.elementToBeClickable(element));
     }
-}
+
```

Comment:
I recommend: make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

```
    protected Actions actions;
    protected WebDriverWait wait;
    protected WebDriver driver;
```